packages/babel-preset-css-prop/src/index.js

```diff
@@ -8,14 +8,10 @@ let pragmaName = '___EmotionJSX'
 // this means if @babel/plugin-transform-react-jsx adds more options, it'll just work
 // and if babel-plugin-emotion adds more options we can add them since this lives in
 // the same repo as babel-plugin-emotion
-export default ({
-  pragma,
-  sourceMap,
-  autoLabel,
-  labelFormat,
-  instances,
-  ...options
-}) => {
+export default (
+  api,
+  { pragma, sourceMap, autoLabel, labelFormat, instances, ...options }
```

Comment:
is this guaranteed to always be an object?

Reply:
`babel-preset-react` seems to assume that: https://github.com/babel/babel/blob/master/packages/babel-preset-react/src/index.js#L10 :)

Reply:
Or actually they use this `declare` helper. Can we use that here also? 🤔 https://github.com/babel/babel/blob/master/packages/babel-helper-plugin-utils/src/index.js#L1

Reply:
Or just default the argument to `= {}`?